qemu/tests/balloon_memhp.py

```diff
@@ -0,0 +1,91 @@
+import random
+import logging
+
+from virttest import utils_test
+from virttest import error_context
+from virttest import utils_numeric
+from virttest.utils_test.qemu import MemoryHotplugTest
+from qemu.tests.balloon_check import BallooningTestWin
+from qemu.tests.balloon_check import BallooningTestLinux
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Balloon and memory hotplug test, i.e. guest-stats-polling-interval test
+    1) boot a guest with balloon device
+    2) enable and check driver verifier in guest(only for windows guest)
+    3) install balloon service in guest(only for windows guest)
+    4) evict balloon
+    5) hotplug memory to guest
+    6) check balloon and guest memory
+    7) enlarge balloon
+    8) unplug the memory in step 5 (only for linux guest)
+    9) check balloon and guest memory
+    10) uninstall balloon service and clear driver verifier(only for
+       windows guest)
+    """
+    def check_memory(op="hotplug"):
+        """
+        Check guest memory after doing operation op
+        :param op: the operation done to guest before
+        """
+        if params['os_type'] == 'windows':
+            memhp_test.check_memory(vm)
+        else:
+            if op == "hotplug":
+                expected_mem = new_mem + mem_dev_sz
+            else:
+                expected_mem = new_mem - mem_dev_sz
+            guest_mem_size = memhp_test.get_guest_total_mem(vm)
+            threshold = float(params.get("mem_threshold", 400))
+            if expected_mem - guest_mem_ size > threshold:
```

Comment:
typo: `guest_mem_ size` ==> `guest_mem_size`

Reply:
@PaulYuuu Thanks for your review,  will update  after confirming with dev about some qemu issues.